docs/documentation/development/bindings/thing-definition.md

```diff
@@ -171,48 +171,7 @@ The user interface can use these values to render labels for values or to provid
 
 ### Channel Categories
```

Comment:
As you have removed the content, I don't think that a headline still makes sense here. Rather check how the information can be put within the existing docs on this page (or any other).